### ISSUE TITLE
JsonSerializer `use` method out of date?

### ISSUE THREAD

**Issue by einverne** [oblac/jodd-site#30]
I am using:

```
    <dependency>
      <groupId>org.jodd</groupId>
      <artifactId>jodd-json</artifactId>
      <version>5.1.4</version>
    </dependency>
```
I found there is no method called `use` now. I have not dug into the documents and code for now.

	String json = new JsonSerializer()
			.use("birthdate", new DateJsonSerializer() {
				@Override
				public void serialize(JsonContext jsonContext, Date date) {
					jsonContext.writeString(dateFormat.format(date));
				}
			})
			.serialize(foo);

**Comment by neroux** [comment]
You are referring to https://github.com/oblac/jodd-site/blob/d4cb14582393d3319effb458b9fbb172d1734472/content/json/2%2Bjson-serializer.md, right?

Should be addressed with #32.

**Comment by igr** [comment]
Thanx @neroux for the quick fix!

Sorry @einverne, trying to migrate things bit by bit to a new version and better, updated documentation. JSON is very next one :)
